packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -76,6 +68,15 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
                 }
 
                 return pluginUtils.addExtension(normalizedPath, ext);
+            } else {
+                try {
+                    return lwcResolver.resolveModule(importee, importer, {
+                        modules: customResolvedModules,
+                        rootDir: customRootDir,
+                    }).entry;
+                } catch {
+                    // noop: let rollup plugins keep resolving
```

Comment:
While adding more test I realized that we should not swallow all the errors, but only the cases where the module is not found. All the other errors (invalid parameters, invalid LWC config, invalid resolution) should throw to surface this to the user. See how the `rollup/node-resolve` plugin is doing this: https://github.com/rollup/plugins/blob/bf94a85b488e2506f10ac64c46d53e70f037aba7/packages/node-resolve/src/util.js#L180

Reply:
That's a good call. I agree.

Reply:
Should I add a pull request that implements something similar to `rollup/node-resolve`?

Reply:
I have a WIP PR: https://github.com/salesforce/lwc/pull/1773.